site/content/docs/commands/env-init.en.md

```diff
@@ -29,26 +29,44 @@ Import Existing Resources Flags
       --import-vpc-id string             Optional. Use an existing VPC ID.
 
 Configure Default Resources Flags
-      --override-private-cidrs strings   Optional. CIDR to use for private subnets (default 10.0.2.0/24,10.0.3.0/24).
-      --override-public-cidrs strings    Optional. CIDR to use for public subnets (default 10.0.0.0/24,10.0.1.0/24).
-      --override-vpc-cidr ipNet          Optional. Global CIDR to use for VPC (default 10.0.0.0/16).
+      --override-az-names strings        Optional. Availability Zone names.
+                                         (default 2 random AZs)
+      --override-private-cidrs strings   Optional. CIDR to use for private subnets.
+                                         (default 10.0.2.0/24,10.0.3.0/24)
+      --override-public-cidrs strings    Optional. CIDR to use for public subnets. 
+                                         (default 10.0.0.0/24,10.0.1.0/24)
+      --override-vpc-cidr ipNet          Optional. Global CIDR to use for VPC.
+                                         (default 10.0.0.0/16)
 
-Global Flags
-  -a, --app string   Name of the application.
+Telemetry Flags
```

Comment:
Boss what do you think about "Observability" 👀 ?

Reply:
ahaha yea, I want to rename it to that but the field in SSM and that appears in `copilot env init --json` is `"telemetry"` in v1.14 💀 so I don't think we can